src/grisp_tools_step.erl

```diff
@@ -112,6 +113,21 @@ collect(#{platform := Platform} = S0) ->
     S2 = event(S1, [{hash, Hash, HashIndex}]),
     mapz:deep_put([build, hash], #{value => Hash, index => HashIndex}, S2).
 
+toolchain(S0) ->
+    ToolchainRoot = mapz:deep_get([paths, toolchain], S0),
+    [error({toolchain_root_invalid, ToolchainRoot}) || not filelib:is_dir(ToolchainRoot)],
+    FILES = [
```

Comment:
Standard in Erlang is to use camel case variable names. Can you change this to `Files` and `File` below?

Reply:
Mmm yep, I forgot this one, i fix